packages/configraph/src/graph.ts

```diff
@@ -137,46 +145,6 @@ export class Configraph<
         if (!entity.parentId) throw new Error('Expected non-root entity to have a parent id set');
       }
     }
-
-    // add graph edges based on "pick"
-    // we will not process individual items yet, but this will give us a DAG of entity dependencies
-    for (const entity of entitiesArray) {
```

Comment:
no longer doing all these shenanigans with pick up front. instead we create the nodes normally, and then do a pass to re-connect picked nodes to their sources